themes/default/content/docs/guides/organizing-projects-stacks/_index.md

```diff
@@ -108,4 +108,72 @@ Stacks have associated metadata in the form of name/value tags. You can assign c
 
 ## Examples
 
+### Monorepo with base infrastructure project
+
+In practice, it's common to have a combination of approaches. Let's build an example of an organizational setup that leverages several different approaches to provide the most functionality and flexibility possible.
```

Comment:
These two sentences kind of say the same thing.

Reply:
I fixed this!